src/library/librarycontrol.cpp

```diff
@@ -172,6 +173,16 @@ LibraryControl::LibraryControl(Library* pLibrary)
             });
 #endif
 
+    // Pure trigger control. Alternative for signal/slot since widgets that want
+    // to call refocusPrevLibraryWidget() are cumbersome to connect to.
+    // This CO is never actually set or read so the value just needs to be not 0
+    m_pRefocusPrevWidgetCO = std::make_unique<ControlPushButton>(
+            ConfigKey("[Library]", "refocus_prev_widget"));
```

Comment:
This should work when connecting to keyboard or controllers. Probably not a use case, but this would make it complete. 
```suggestion
            ConfigKey("[Library]", "refocus_prev_widget"));
    m_pRefocusPrevWidgetCO->setButtonMode(ControlPushButton::TRIGGER);
```

Reply:
Interesting! Will try.

Reply:
Jup, works. Thanks.